Gemfile

```diff
@@ -171,7 +171,8 @@ group :test, :development do
   gem 'rspec-rails', '~> 3.5.0'
   gem 'pry', '~> 0.10.0'
   gem 'pry-debugger', '~> 0.2.0', :platforms => :ruby_19
-    gem 'public_suffix', '~> 1.4.0', '< 1.5.0'
+  gem 'public_suffix', '~> 1.4.0', '< 1.5.0'
```

Comment:
This was an intentional indent as we don't use `public_suffix` directly. Pry depends on it, but we need to pin the specific version for compatibility reasons.